udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -103,8 +103,9 @@ <h1 class="fr-mb-5v fr-h3">
             </h1>
             <h2 id="description" class="subtitle fr-mb-1w">{{ _('Description') }}</h2>
             <read-more max-height="description" class="fr-mb-n1w">
-                {{ dataset.description|markdown }}
+                <markdown-viewer data-content="{{dataset.description|to_json}}"></markdown-viewer>
```

Comment:
I think it's better to do a simple escape the HTML attribute (if it's possible, I didn't test it but I don't see why it wouldn't work… See https://stackoverflow.com/questions/9187946/escaping-inside-html-tag-attribute-value
```suggestion
                <markdown-viewer content="{{dataset.description|replace('"', '&quot;')}}"></markdown-viewer>
```
This allow to remove all the `ref` and `onMounted` inside the component and only keep a single props `content`.

Reply:
We tried to do it with a `props` for the Search component and this is indeed simpler for us. But the unintended side-effect is that an invalid props will break vue and the component will never mount. I'll try it when we redo it to use the same parser as the markdown editor.